.circleci/test-deploy.yml

```diff
@@ -187,6 +187,52 @@ jobs:
             grep -v simpleTest test-results/matlab/filterdtagresult.xml
           shell: bash
 
+      - run:
+          name: Set up diary for logging
+          command: |
+            echo 'diary console.log' >> startup.m
+          shell: bash
+      - matlab/run-tests:
+          strict: true
+          source-folder: src
+      - run:
+          name: Verify tests ran with strict checks
+          command: |
+            set -e
+            grep -q "runner.addPlugin(FailOnWarningsPlugin())" console.log
+            rm console.log
+          shell: bash
+      - matlab/run-tests:
+          use-parallel: true
+          source-folder: src
+      - run:
+          name: Verify tests ran in parallel
+          command: |
+            set -e
+            grep -q "Unable to get or start a parallel pool. Running tests in serial instead." console.log
```

Comment:
By "get", do you mean "access"? If so, please replace with "access".

Reply:
Here "get" is the actual term generated by the script. For this, I guess we'll need to modify genscript.

Reply:
Could just look for "parallel pool" because this may work in the not too distant future.

Reply:
Updated, thanks!